index.html

```diff
@@ -73,8 +73,99 @@
       </div>
     </div>
   </header>
-  <main></main>
+
+    <main>
+     <section class="product">
+      <div class="carousel">
+      <div class="product-carousel">
+        <div class="carousel-img">
+          <div class="carousel-item product-banner" style="background-image: url('image/Banner.jpg')"></div>
+          <div class="carousel-item product-banner" style="background-image: url('image/Banner.jpg')"></div>
+          <div class="carousel-item product-banner" style="background-image: url('image/Banner.jpg')"></div>
+        </div>
+      <div class="indicators">
+        <span class="dot" onclick="currentSlide(0)"></span>
+        <span class="dot" onclick="currentSlide(1)"></span>
+        <span class="dot" onclick="currentSlide(2)"></span>
+      </div>
+     </div>
+     <button class="prev" onclick="moveSlide(-1)">&#10094;</button>
+     <button class="next" onclick="moveSlide(1)">&#10095;</button>
+     </div>
+
+      <div class="product-derecha">
+        <div class="product-card" >
+          <div class="img-product">
+            <img src="image/1.png" alt="" class="rating">
+            <span class="star">&#9733;</span>
+            <span class="rating-number">5</span>
+          </div>
+
+          <div class="sales-products">
+            <span class="product-categoria">HOME ELECTRICS</span>
+            <span class="product-nombre">Aire acondicionado 12.000 BTU</span>
+            <div class="product-valor">
+            <span class="product-total"><p>159.99</p></span>
```

Comment:
Los `p` no van dentro de los `span`. Los elementos de bloque (`p`, `div`, etc) no van dentro de elementos inline (`a`, `span`, `button`, etc)
```suggestion
            <p class="product-total">159.99></p>
```